src/modules/siptrace/doc/siptrace_admin.xml

```diff
@@ -656,6 +654,11 @@ hlog("$hdr(P-MyID)", "Another one with a custom correlation ID");
 	Stateless forwarded messages (forward()) are not logged if you set the
 	flag, use sip_trace() inside <emphasis>onsend_route</emphasis> block.
 	</para>
+	<para>
+	If dialog level tracing is used internally generated BYE transaction(in
+	case of internal timeouts) won't be traced. At the moment kamailio offers
+	no posibility to trace those messages.
```

Comment:
Actually it should be possible to capture these BYEs, as we do for ACK, in onsend_route or in tm's module local_route.

Reply:
Thanks for mentioning. I've found callbacks for these BYEs but inside them the message is not parsed, I can only access the char buffer. There's no sip_msg structure created, therefore I needed to parse it in order to trace it. I didn't know whether it would be a good idea to parse the message just for it to be traced since SIP message parsing is an expensive operation. What are your thoughts about this?

Reply:
Daniel commented in the main comment thread about the sip parsing, that it is actually not that expensive anymore